src/borrow_tracker/tree_borrows/diagnostics.rs

```diff
@@ -105,7 +105,7 @@ impl TbError {
         err_tb_ub(match self.error_kind {
             CannotAccess(perm) => {
                 format!(
-                    "Impossible to perform a {is_foreign} {kind} access through {initial:?} because {current:?} has permissions {perm:?}.",
+                    "Impossible to perform a {is_foreign} {kind} through {initial:?} because {current:?} has permissions {perm:?}.",
```

Comment:
I think our messages usually start lower-case (check the code elsewhere in Miri)